source/transactions/tests/retryable-abort-errorLabels.yml

```diff
@@ -0,0 +1,127 @@
+runOn:
+    -
+        # TODO: Update once SERVER-41245 is released
+        minServerVersion: "4.3.x"
+        topology: ["replicaset, sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+tests:
+  - description: abortTransaction only retries once after RetryableWriteError from server
```

Comment:
"with RetryableWriteError label from server" may read better. If you agree, feel free to make a similar change to other tests that used this original language. I think that's at least commitTransaction but I'm not sure if other tests copied as well.